q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,278 @@
+
+* A solid track record of being constructive and helpful
+* Significant contributions to the project's goals, in any form
+* Willingness to dedicate some time to improving Perl
+
+Contributions are not limited to code. Here is an incomplete list of areas
+where contributions may be considered for joining the Core Team:
+
+* Working on community management and outreach
+* Providing support on mailing lists, IRC, or other forums
+* Triaging tickets
+* Writing patches (code, docs, or tests)
+* Reviewing patches (code, docs, or tests)
+* Participating in design decisions
+* Providing expertise in a particular domain (security, i18n, etc.)
+* Managing Perl infrastructure (websites, CI, documentation, etc.)
+* Maintaining significant projects in the Perl ecosystem
+* Creating visual designs
+
+Core Team membership acknowledges sustained and valuable efforts that align
+well with the philosophy and the goals of the Perl project.
+
+### Term
+
+Core Team members serve until they are removed.
+
+### Removal
+
+Core Team Members may resign their position at any time.
+
+In exceptional circumstances, it may be necessary to remove someone from the
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+### Inactivity
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves "inactive". Inactive members do not nominate or vote.  Inactive
+members may declare themselves active at any time, except when a vote has been
+proposed and is not concluded.  Eligibility to nominate or vote will be
+determined by the Vote Administrator.
+
+To record and honor their contributions, inactive Core Team members will
+continue to be listed alongside active members.
+
+### No Confidence in the Steering Council
+
+The Core Team may remove either a single Steering Council member or the entire
+Steering Council via a No Confidence Vote.
+
+A No Confidence Vote is triggered when a Core Team member calls for one
+publicly on an appropriate project communication channel, and another Core
+Team member seconds the proposal.
+
+If a No Confidence Vote removes all Steering Council members, the Vote
+Administrator of the No Confidence Vote will then administer an election
+to select a new Steering Council.
+
+### Amending Perl Rules of Governance
+
+Any Core Team member may propose amending the Perl Rules of Governance by
+sending a proposal to the Steering Council.  The Steering Council must decide
+to approve or reject the proposal.  If approved, the Steering Council will
+administer an Amendment Vote.
+
+### Rules for Voting
```

Comment:
One thing that's missing here is a timeline for votes. For things like membership changes and no confidence votes, those need to happen relatively quickly (maybe within 2 weeks of being proposed?). Amendments can probably take longer.

Maybe there's a mechanism for voting to continue a discussion pas the initial deadline?

Reply:
See lines 150-151.

Reply:
Doh! No idea how I missed that.